.github/workflows/mayhem-api.yml

```diff
@@ -0,0 +1,97 @@
+name: "Mayhem API analysis"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Given that we can run 50 jobs (in my account) I think we should run this once per day just on the main branch leaving 20 jobs left unused? Also I didn't think about the secret on forks yet, so if this can even run if someone opens a PR from their fork.